packages/react/src/SteppedUpContent.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactNode, useEffect, useRef } from 'react';
+
+import { useStepUp, useIsSteppedUp, useIsAuthenticated } from '@frontegg/react-hooks';
+
+export interface SteppedUpProps {
+  maxAge?: number;
+  render?: (isSteppedUp: boolean) => React.ReactNode | null;
+  children?: ReactNode;
```

Comment:
We can also add `callback` function as a prop and call it if user is stepped up

Reply:
can't see how.. not familiar with a pattern like that. I think that render should be used when you need a callback